Patches/CheckGameEndPatch.cs

```diff
@@ -491,7 +491,7 @@ public bool CheckGameEndByLivingPlayers(out GameOverReason reason)
         {
             reason = GameOverReason.ImpostorByKill;
 
-            if (CustomRolesHelper.RoleExist(CustomRoles.Sunnyboy) && Main.AllAlivePlayerControls.Count() > 1) return false;
+            if (CustomRolesHelper.RoleExist(CustomRoles.Sunnyboy) && Main.AllAlivePlayerControls.Any()) return false;
```

Comment:
This needs to be reverted cause count > 1 isn't the same as Any.
> 1 means 2 or more, while Any means 1 or more

dw I ran into the same problem 🤣

Reply:
Could we not just do >= 1?

Reply:
No cause it has to check if there's anyone else other than the sunnyboy alive, aka the player count is higher than 1 or not

Reply:
Yes, I'll fix it, I didn't pay attention to it at the time

Reply:
Fixed